### DISCUSSION TITLE
Best Practice of Figma for Indie Hackers - Rene's Official Website

### DISCUSSION THREAD

**Discussion by giscus** [RiverTwilight/rene.wang#43] [Announcements]
# Best Practice of Figma for Indie Hackers - Rene's Official Website

If you're an indie like me, in this post, I'll share how I manage my team, files, and pages in Figma. This is tailored for solo makers and will boost your work efficiency.

http://localhost:3000/en-US/p/best-practice-of-figma-for-indie-hackers

**Comment by rivertwilight** [comment]
Hello World

**Comment by rivertwilight** [comment]
Good
